test/src/test_board.py

```diff
@@ -0,0 +1,51 @@
+from unittest import TestCase
+
+from board import Board
+
+
+# TODO: parameterized with different levels, something like this: https://stackoverflow.com/questions/32899/how-do-you-generate-dynamic-parameterized-unit-tests-in-python
+# https://docs.python.org/3/library/unittest.html
+class TestBoard(TestCase):
+
+    def setUp(self):
+        self.board = Board('/Users/brookeryan/PycharmProjects/CS271/test/input/levels/level1.txt')
+        self.height = 11
+        self.width = 19
+        self.dimensions = (self.height, self.width)
+
+    # Simply checks to make sure the board and its attributes are set
+    def test_board_not_None(self):
+        self.assertIsNotNone(self.board)
+        self.assertIsNotNone(self.board.__getattribute__('height'))
+        self.assertIsNotNone(self.board.__getattribute__('width'))
+        self.assertIsNotNone(self.board.__getattribute__('board'))
+
+    # Tests that the dimensions of the generated board are correct
+    def test_dimensions(self):
```

Comment:
@jcness216 this test is failing, I think the height and width in your board.py file got mixed up